


<html>
<head>
<title>Glutamyl-tRNA Synthetase (Eukaryote-like)</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>

<script async src="https://www.googletagmanager.com/gtag/js?id=G-KBC488H8EG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KBC488H8EG');
</script>


<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>


						The eukaryote-like glutamyl-tRNA synthetase (GluRS-E) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of the amino acid glutamate to its cognate tRNA:

					<div class="reaction">
						<span>
							$ \text{Glu} + \text{tRNA}^\text{Glu} + \text{ATP} \xrightarrow{\text{GluRS-E}} \text{Glu-tRNA}^\text{Glu} + \text{AMP} + \text{PP}_i  $
						</span>
					</div>


					As discussed by Hadd and Perona 2014, there is a complex coevolutionary history between glutamyl- and glutaminyl-tRNA syntheteases, which comprise subclass Ib (Perona and Hadd. 2012, Gomez et al., 2020). 
					Their diversification occurred after the last universal common ancestor, with bacterial-like forms being characterised by an $\alpha$-helical anticodon binding domain, and the archael and eukaryotic forms possessing a $\beta$-barrel anticodon binding domain.
					While many contemporary systems express both GlnRS and GluRS, their ancestor was most likely a non-discriminating form, which would attach Glu to tRNA<sup>Gln</sup>.
					A second enzymic step, performed by an amidotransferase, would correct the misacylated tRNA prior to protein synthesis, as it does with <a href="/class2/asp2/">AsxRS</a> (Lapointe et al. 1986, Raczniak et al. 2001).
					This non-discriminating enzyme is still found in systems which lack GlnRS, such the archaea, which express <a href="/class1/glu2">GlxRS-A</a>, as well as certain bacteria which have a non-discriminating variant <a href="/class1/glu1">GlxRS-B</a>, or a noncognate variant GluGlnRS which attaches Glu to tRNA<sup>Gln</sup> (Salazar et al. 2003, Skouloubris et al. 2003).
					Is is likely that GlnRS originated in the eukaryota, and was later acquired by certain bacteria through horizontal gene transfer (Siatecka et al. 1998). <br><br>

						<br><br>


					This page details the eukaryote-like GluRS-E, which is a discriminating form of GluRS. 
					It is found in many eukaryotic cytosols, and is characterised by an N-terminal glutathione S-transferase (GST) domain (Hadd and Perona. 2014).
					Chung et al. 2022 identified four potential regions of the <i>Gallus Gallus</i> GluRS-E which may offer selectivity against tRNA<sup>Gln</sup>.
					<br><br>
					


				The N-terminal catalytic domain of GluRS-E closely resembles the other members of subclass Ib: <a href="/class1/gln">GlnRS</a>, <a href="/class1/glu1">GluRS-B</a>, GlxRS-A,  and GlxRS-B.
					Their catalytic domains are characterised by an insertion within CP1, containing a loop flanked by two helices (SC1b IM), which 
					may play a role in acceptor stem recognition  (Rath et al. 1998, Nureki et al. 2010).
					GluRS-E has a <a href="/superfamily/class1/Anticodon_binding_domain_Q">$\beta$-barrel anticodon binding domain</a> located at the C-terminal end (Rould et al. 1991), which is homologous to GlxRS-A and GlnRS.
					However it is distinct to the bacterial form GluRS-B, which instead has an <a href="/superfamily/class1/Anticodon_binding_domain_E">$\alpha$-helical anticodon binding domain</a> of bacterial origin. 
					The members of subclass Ib, alongside LysRS-I and <a href="/class1/arg">ArgRS</a>, require the presence of tRNA to catalyse activation of the amino acid substrate (Dubois et al. 2005).
					<br><br>

			A fused glutamyl-prolyl tRNA synthetase (EPRS) also exists in most animals (Berthonneau et al. 2000). 
					EPRS contains the catalytic domains from both GluRS-E and <a href="/class2/pro1/">ProRS-A</a>, as well as the ProRS-A zinc binding domain. 
					Glutamic acid is a metabolic precursor for proline, which suggests a possible explanation for the origin of this protein (Eswarappa et al. 2018). <br><br>



				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">

				<div>
				Daniel Y. Dubois, Jacques Lapointe and Shun-ichi Sekine. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 10: Glutamyl-tRNA Synthetases.
			</div>
				
	
			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>


			<div>
				Rath, Virginia L., et al. "How glutaminyl-tRNA synthetase selects glutamine." Structure 6.4 (1998): 439-449.
			</div>

			<div>
				Chung, Scisung, et al. "Regulation of BRCA1 stability through the tandem UBX domains of isoleucyl-tRNA synthetase 1." Nature Communications 13.1 (2022): 6732.
			</div>

				<div>
				Eswarappa, Sandeep M., et al. "Metabolic origin of the fused aminoacyl-tRNA synthetase, glutamyl-prolyl-tRNA synthetase." Journal of Biological Chemistry 293.49 (2018): 19148-19156.
			</div>




			<div>
				Mailu, Boniface M., et al. "A nondiscriminating glutamyl-tRNA synthetase in the Plasmodium apicoplast: the first enzyme in an indirect aminoacylation pathway." Journal of Biological Chemistry 288.45 (2013): 32539-32552.
			</div>



			<div>
				Hadd, Andrew, and John J. Perona. "Coevolution of specificity determinants in eukaryotic glutamyl-and glutaminyl-tRNA synthetases." Journal of molecular biology 426.21 (2014): 3619-3633.
			</div>



			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>

		<div>
				Siatecka, Miroslawa, et al. "Modular evolution of the Glx‐tRNA synthetase family: Rooting of the evolutionary tree between the bacteria and archaea/eukarya branches." European Journal of Biochemistry 256.1 (1998): 80-87.
			</div>
		
		</div>


	</div>
</body>
</html>
