


<html>
<head>
<title>aaRS</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/javascript"
  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>



<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">

		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>




					The mitochondrial	phenylalanyl-tRNA synthetase (PheRS-M) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of the amino acid phenylalanine to its cognate tRNA:



					<div class="reaction">
						<span>
							$ \text{Phe} + \text{tRNA}^\text{Phe} + \text{ATP} \xrightarrow{\text{PheRS-M}} \text{Phe-tRNA}^\text{Phe} + \text{AMP} + \text{PP}_i  $
						</span>
					</div>

					The gene for PheRS-M is expressed in the nucleus and the protein is localised to the mitochondria and chloroplasts (Duchêne et al., 2005). 
					The phenylalanyl-tRNA synthetases (PheRS) are considered to be one of the most evolutionarily complex enzymes among aminoacyl-tRNA synthetases (Klipcan et al., 2010). 
					There are three variants of PheRS, with PheRS-M being the only monomeric member of the class II superfamily. 
					The other two variants are heterotetrameric, with one <a href="../phe1">variant</a> found primarily in bacteria and the <a href="../phe3">variant</a> in eukaryotes/archaea (Finarov et al., 2010). <br><br>


					The catalytic core of PheRS-M is similar to that of the other PheRS families, as well as <a href="../sep">SepRS</a> and <a href="../pyl">PylRS</a>. 
					Together, these enzymes constitute subclass IIc (Kavran et al. 2007, Perona et al. 2012, Valencia-Sánchez et al. 2016, Gomez et al. 2020). 
					The core of PheRS-M is most similar to the &alpha; bacterial subunit <a href="../phe1">PheRS-B&alpha;</a>.
					 In contrast, the C-terminal anticodon binding domain appears to have been borrowed from the bacterial &beta; subunit <a href="../phe2">PheRS-B&alpha;</a> (domain B8; Klipcan et al. 2010). This suggests that the mitochondrial PheRS may be a chimera between the two bacterial PheRS subunits. <br><br>


					 Unlike the cytosolic variants, there is no known editing activity for the mitochondrial PheRS (Roy et al. 2005, Klipcan et al. 2010). All types of PheRS are known to mischarge tyrosyl groups, which suggests that mitochondrial translation of phenylalanine may be less accurate than the cytosolic form.



			
				</div>
				
			</li>
			
		</ul>

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>




		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">


			<div>
				Roy, Hervé, et al. "Loss of editing activity during the evolution of mitochondrial phenylalanyl-tRNA synthetase." Journal of Biological Chemistry 280.46 (2005): 38186-38192.
			</div>

			<div>
					Klipcan, Liron, et al. "Structural aspects of phenylalanylation and quality control in three major forms of phenylalanyl-tRNA synthetase." Journal of amino acids 2010 (2010).
				</div>


				<div>
					Finarov, Igal, et al. "Structure of human cytosolic phenylalanyl-tRNA synthetase: evidence for kingdom-specific design of the active sites and tRNA binding patterns." Structure 18.3 (2010): 343-353.
				</div>


		<div>
			Duchêne, Anne-Marie, et al. "Dual targeting is the rule for organellar aminoacyl-tRNA synthetases in Arabidopsis thaliana." Proceedings of the National Academy of Sciences 102.45 (2005): 16484-16489.
		</div>

			<div>
				Klipcan, Liron, et al. "The tRNA-induced conformational activation of human mitochondrial phenylalanyl-tRNA synthetase." Structure 16.7 (2008): 1095-1104.
			</div>

			<div>
				Kavran, Jennifer M., et al. "Structure of pyrrolysyl-tRNA synthetase, an archaeal enzyme for genetic code innovation." Proceedings of the National Academy of Sciences 104.27 (2007): 11268-11273.
			</div>


				<div>
					Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.
			</div>


			<div>
				Safro, Mark. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 22: Phenylalanyl-tRNA Synthetases.
			</div>

			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>

		

			<div>
				Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.
			</div>



			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>

		</div>


	</div>
</body>
</html>
