


<html>
<head>
<title>aaRS</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>




<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">

		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>


						Phenylalanyl tRNA synthetase (PheRS) is an enzyme which attaches phenylalanine to its cognate tRNA: <br><br>


					<ul>
						<li>Phe + tRNA<sup>Phe</sup> + ATP &rightarrow; Phe-tRNA<sup>Phe</sup> + AMP + PP<sub>i</sub> </li>
					</ul>


					PheRS is probably the most complex of amino acyl tRNA synthetases (Klipcan et al. 2010). While most members of the class II superfamily are homodimeric, this enzyme operates as a heterotetramer &alpha;<sub>2</sub>&beta;<sub>2</sub>. 
					In total, there are three known variants of PheRS, and a total of five class II families (Finarov et al. 2010).
					Two of these variants are heterotetrameric - one predominantly found in bacteria (b-PheRS) and one in eukaryotes/archaea (e-PheRS) - while the other variant is monomeric - and found in mitochondria.
					This page details the &beta; subunit of the eukaryotic variant. Also see the <a href="../phe3">eukaryotic &alpha; subunit</a>, the <a href="../phe1">bacterial &alpha; subunit</a>, the  <a href="../phe2">bacterial &beta; subunit</a>, and the <a href="../phe5">mitochondrial PheRS</a>. <br><br>



						The C-terminal catalytic core of &beta; is similar to that of the other PheRS familes, as well as <a href="../sep">SepRS</a> and <a href="../pyl">PylRS</a>. Collectivey, these members constitute subclass IIc (Kavran et al. 2007, Perona et al. 2012, Valencia-Sánchez et al. 2016, Gomez et al. 2020). <br><br>


								Although both eukaryotic subunits possess a class II catalytic domain, catalytic activity appears to be restricted to the &alpha; subunit. tRNA recognition also occurs in the &alpha; chain (in contrast to the bacterial tetramer), while editing activity is restricted to &beta;. <br><br>


						The catalytic domain does not appear to possess catalytic activity (Fishman et al. 2001), but still contains the six antiparallel strands characteristic of a class II aaRS. As such, the sequence at the stuctural positions of motifs 1, 2, and 3 do not follow the motif patterns. The positions one would normally expect to see these motifs are labelled for convenience.  <br><br>



						The editing domain (hB3 and hB4) is located near the N-terminal and resembles that of the <a href="../phe2">bacterial &beta; PheRS subunit</a> (Finarov et al. 2010), which is known to target mischarged tyrosines (Roy et al. 2004).
						






				</div>
				
			</li>
			
		</ul>



		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		
			<div id="references">


			<div>
					Klipcan, Liron, et al. "Structural aspects of phenylalanylation and quality control in three major forms of phenylalanyl-tRNA synthetase." Journal of amino acids 2010 (2010).
				</div>


				<div>
					Finarov, Igal, et al. "Structure of human cytosolic phenylalanyl-tRNA synthetase: evidence for kingdom-specific design of the active sites and tRNA binding patterns." Structure 18.3 (2010): 343-353.
				</div>

				<div>
					Fishman, Roman, et al. "Structure at 2.6 Å resolution of phenylalanyl-tRNA synthetase complexed with phenylalanyl-adenylate in the presence of manganese." Acta Crystallographica Section D: Biological Crystallography 57.11 (2001): 1534-1544.
				</div>

			<div>
				Kavran, Jennifer M., et al. "Structure of pyrrolysyl-tRNA synthetase, an archaeal enzyme for genetic code innovation." Proceedings of the National Academy of Sciences 104.27 (2007): 11268-11273.
			</div>

			<div>
				Cusack, Stephen, Michael Härtlein, and Reuben Leberman. "Sequence, structural and evolutionary relationships between class 2 aminoacyl-tRNA synthetases." Nucleic acids research 19.13 (1991): 3489-3498.
			</div>

			<div>
					Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.
			</div>


				<div>
				Safro, Mark. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 22: Phenylalanyl-tRNA Synthetases.
			</div>

			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>

		



			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>


							<div>
					Roy, Herve, et al. "Post‐transfer editing in vitro and in vivo by the β subunit of phenylalanyl‐tRNA synthetase." The EMBO journal 23.23 (2004): 4639-4648.
				</div>


		
		</div>



	</div>
</body>
</html>
