


<html>
<head>
<title>aaRS</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/javascript"
  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>




<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">

		
		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>



				Pyrrolysyl-tRNA synthetase (PylRS) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of the amino acid pyrrolysine to its cognate tRNA:



					<div class="reaction">
						<span>
							$ \text{Pyl} + \text{tRNA}^\text{Pyl} + \text{ATP} \xrightarrow{\text{PylRS}} \text{Pyl-tRNA}^\text{Pyl} + \text{AMP} + \text{PP}_i  $
						</span>
					</div>




				Pyrrolysine is encoded by the stop codon UAG in some archaea and bacteria (Srinivasan et al. 2002), which is achieved through a specialized molecule tRNA<sup>Pyl</sup> that recognizes the UAG stop codon. The domain architecture of PylRS differs between archaea and bacteria (Srinivasan et al. 2002). The archaeal PylRS contains two domains: an N-terminal domain involved in tRNA recognition, and a C-terminal catalytic domain, which is also involved in tRNA binding. The full-length archaeal PylRS was found to be quite insoluble (Suzuki et al. 2017). In contrast, the bacterial PylRS is encoded by two genes, pylSn (encoding the N-terminal domain), and pylCn (encoding the C-terminal domain). The latter protein is dimeric (Lee et al. 2008), but the full bacterial enzyme has not been studied as extensively as the archaeal enzyme, and likely exists as a heterotetramer.  <br><br>

				The catalytic domain of PylRS is similar to that of <a href="../sep">SepRS</a> and the five PheRS families, which collectively constitute subclass IIc (Kavran et al. 2007, Perona et al. 2012, Valencia-Sánchez et al. 2016, Gomez et al. 2020). The catalytic domain is typical among class II enzymes, containing six antiparallel strands. The binding of ATP is coordinated by the arginine tweezers, located in motifs 2 and 3 (Kaiser et al. 2018). The N-terminal domain binds the T-loop and variable loop of tRNA<sup>Pyl</sup> (Suzuki et al. 2017) in contrast with many amino acyl tRNA synthetases which instead recognize the anticodon. It appears that PylRS lacks editing activity (Gomez et al. 2020).

			
				</div>
				
			</li>
			
		</ul>
		


		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>




		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">
			<div>
				Kavran, Jennifer M., et al. "Structure of pyrrolysyl-tRNA synthetase, an archaeal enzyme for genetic code innovation." Proceedings of the National Academy of Sciences 104.27 (2007): 11268-11273.
			</div>

			<div>
				Suzuki, Tateki, et al. "Crystal structures reveal an elusive functional domain of pyrrolysyl-tRNA synthetase." Nature chemical biology 13.12 (2017): 1261-1266.
			</div>

			<div>
				Lee, Marianne M., et al. "Structure of Desulfitobacterium hafniense PylSc, a pyrrolysyl-tRNA synthetase." Biochemical and biophysical research communications 374.3 (2008): 470-474.
			</div>

			<div>
				Srinivasan, Gayathri, Carey M. James, and Joseph A. Krzycki. "Pyrrolysine encoded by UAG in Archaea: charging of a UAG-decoding specialized tRNA." Science 296.5572 (2002): 1459-1462.
			</div>




				<div>
					Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.
			</div>


			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>

		

			<div>
				Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.
			</div>



			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>

		</div>


	</div>
</body>
</html>
