


<html>
<head>
<title>Class II Catalytic domain</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="/css/styles.css">
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script type='text/javascript' src='/bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>




<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS(false, true);
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>

					The catalytic domain of the Class II amino acyl tRNA synthetase (AARS-II) activates amino acids in order to charge their cognate tRNAs. 
					In total, twelve of the twenty two proteinogenic amino acids are charged by AARS-II. 
					These amino acids are alanine, asparagine, aspartate, cysteine, glycine, histidine, lysine, phenylalaline, proline, pyrrolysine, serine, and threonine. 
					Lysine and cysteine are also processed by the Class I amino acyl tRNA synthetases. These twelve amino acids are processed by twenty one distinct AARS-II families, a single representative from each of which is displayed below. <br><br>



					Most AARS-II families are homodimeric. However, <a href="/class2/phe1">PheRS</a> and the bacterial <a href="/class2/gly2">GlyRS</a> exist as heterotetramers, while   <a href="/class2/sep">SepRS</a> is homotetrameric, and the <a href="/class2/phe5">mitochondrial PheRS</a> is monomeric.  <br><br>
					

					Although they are part of the class II catalytic domain superfamily, the &beta; subunits of the <a href="/class2/phe2">bacterial PheRS</a> and <a href="/class2/phe4">archaeal PheRS</a> families do not in fact catalyse amino acid activation (Klipcan et al. 2020). Instead, catalysis in these enzymes is performed by the respective &alpha; subunits <a href="/class2/phe1">PheRS-B&alpha;</a> and <a href="/class2/phe3">PheRS-A&alpha;</a>  .
					As such, these two &beta; chains are lacking the arginine tweezers which would normally coordinate ATP binding; one in motif 2 and the other in motif 3 (Kaiser et al. 2018).  <br><br>



					 The common structure of AARS-II is described by three &alpha;-helices H1-H3, five &beta;-strands S1-S5, and a strand followed immediately by a helix SH1.
					 The catalytic domain itself is a six stranded anti-parallel &beta; fold. The first five strands are denoted by S1-S5, while the sixth strand is denoted by SH1, as it is immediately followed by a helix. There is also a very short &beta;-strand, typically 2-3 residues in length, which resides within motif 1 and runs parallel with S1. The two arginine tweezers reside in S1 and SH1, which are close proximity with one other.
					 The loops which connect these secondary structural elements are quite variable in length and contain a multitude of family- or clade-specific insertion modules (IM). <br><br>



					 Most AARS-II members contain the same core six antiparallel &beta;-strands (plus a short strand parallel to S1).
					 This anti-parallel fold is bounded between S1 (near the N-terminus) and S3 (>50 residues downstream). However, some members contain additional strands which run parallel with S3. 
					 First, the GlyRS-AE insertion module 2 is a small ~40 residue module which contains a &beta;-hairpin. This hairpin runs parallel with S3, and adds an two additional strands to the fold for both dimeric GlyRS families <a href="/class2/gly1">GlyRS-A</a> and <a href="/class2/gly3">GlyRS-E</a>.
					 Second, the subclass 2b insertion module is a ~80 residue module with a short &beta; strand at its N-terminus which runs parallel to S3. This insertion module is found in <a href="/class2/lys">LysRS-II</a>, <a href="/class2/asn">AsnRS</a>, <a href="/class2/asp1">AspRS</a>, and <a href="/class2/asp2">AsxRS</a>. 

					 <!--  TODO: 3. His 4. Ala 5. Thr. Are Ala and Thr related, or is it convergent evolution?-->





				
				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">

			
			<div>
				Wang, Sheng, Jian Peng, and Jinbo Xu. "Alignment of distantly related protein structures: algorithm, bound and implications to homology modeling." Bioinformatics 27.18 (2011): 2537-2545.
			</div>


			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>


			<div>
					Klipcan, Liron, et al. "Structural aspects of phenylalanylation and quality control in three major forms of phenylalanyl-tRNA synthetase." Journal of amino acids 2010 (2010).
				</div>

					<div>
				Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.
			</div>
		
		</div>


	</div>
</body>
</html>
