


<html>
<head>
<title>Class II Catalytic domain</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="/css/styles.css">
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script type='text/javascript' src='/bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>




<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>

<script async src="https://www.googletagmanager.com/gtag/js?id=G-KBC488H8EG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KBC488H8EG');
</script>



<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS(false, true);
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>


					The catalytic domain of the Class II aminoacyl-tRNA synthetase (AARS-II) activates amino acids so that their cognate tRNA can be charged. 
					In total, thirteen of the 22 proteinogenic amino acids are incorporated into the protein, through 21 distinct AARS-II families; a single representative from each of which is displayed below.
					These amino acids are alanine, asparagine, aspartate, cysteine, glycine, histidine, lysine, phenylalaline, proline, pyrrolysine, selenocysteine, serine, and threonine. 
					Lysine and cysteine are also incorporated during translation via the Class I amino acyl tRNA synthetases <a href="/class1/lys">LysRS-I</a> and <a href="/class1/cys">CysRS</a>. 
					 <br><br>



					Most AARS-II families are homodimeric. However, the <a href="/class2/phe1">cytosolic PheRS</a> and the bacterial <a href="/class2/gly2">GlyRS</a> exist as heterotetramers, while   <a href="/class2/sep">SepRS</a> is homotetrameric, and the <a href="/class2/phe5">mitochondrial PheRS</a> is monomeric.  <br><br>
					

					Although they are part of the class II catalytic domain superfamily, the &beta; subunits of the <a href="/class2/phe2">bacterial PheRS</a> and <a href="/class2/phe4">archaeal PheRS</a> families do not in fact catalyse amino acid activation (Klipcan et al. 2020). Instead, catalysis in these enzymes is performed by the respective &alpha; subunits <a href="/class2/phe1">PheRS-B&alpha;</a> and <a href="/class2/phe3">PheRS-A&alpha;</a>  .
					As such, these two &beta; chains are lacking the arginine tweezers which would normally coordinate ATP binding; one in motif 2 and the other in motif 3 (Kaiser et al. 2018).  <br><br>



					The universal AARS-II structure is composed of 3 &alpha;-helices (H1-H3), 5 &beta;-strands (S1-S5) and a strand-helix (SH1). Its catalytic domain is a six-stranded anti-parallel beta fold with the first five strands being denoted as S1-S5 and the sixth strand being part of SH1. There's also a short beta strand within motif 1 that runs parallel with S1. The arginine tweezers are located in S1 and SH1, which are close to each other. The loops connecting these secondary structural elements are variable in length and contain multiple insertion modules (IM). <br><br>



					All AARS-II members have the same six anti-parallel beta strands bound by S1 (near the N-terminus) and S3 (>50 residues downstream), plus the short strand parallel to S1. However, some members have additional strands parallel to S3. 
					First, <a href="/superfamily/class2/GlyRS-AE_IM2">GlyRS-AE insertion module 2</a> is a small (~40 residue) module located between S2 and H3 that contains a &beta;-hairpin running parallel to S3 and adds two additional strands to the fold for both dimeric GlyRS families (<a href="/class2/gly1">GlyRS-A</a> and <a href="/class2/gly3">GlyRS-E</a>). 
					Second, <a href="/superfamily/class2/Subclass_2b_IM_1">subclass 2b insertion module 1</a>, located between H3 and S3, is found in <a href="/class2/lys">LysRS-II</a>, <a href="/class2/asn">AsnRS</a>, <a href="/class2/asp1">AspRS</a>, and <a href="/class2/asp2">AsxRS</a>. It is over 70 residues and contains a short &beta;-strand at its N-terminus which runs parallel to S3. 
					Finally, the <a href="/superfamily/class2/AlaRS_IM">AlaRS</a>, <a href="/superfamily/class2/HisRS_IM">HisRS</a>, and <a href="/superfamily/class2/ThrRS_IM">ThrRS insertion modules</a> are all located between S3 and S4. These three modules are structurally distinct and appear to have arisen independently. They all contain a short &beta;-strand which runs parallel to S3.<br><br>





					A &beta; hairpin is located between motifs 1 and 2 in most members of the AARS-II superfamily, with the exception of subclass 2d (<a href="/class2/ala">AlaRS</a> and <a href="/class2/gly2">GlyRS-B&alpha;</a>). 
					 The second part of the hairpin is located at the N-terminus of the protozyme and is universal, while the first strand is responsible for stabilizing the hairpin, and it's likely to have evolved early in the evolutionary history of AARS-II. The hairpin's loop has since seen the appearance of insertion modules for <a href="/superfamily/class2/SepRS_IM">SepRS</a>, <a href="/superfamily/class2/SerRS-A_IM">SerRS-A</a>, and <a href="/superfamily/class2/PheRS-A&alpha;_IM">PheRS-A&alpha;</a>.

			



				
				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">

			
			<div>
				Wang, Sheng, Jian Peng, and Jinbo Xu. "Alignment of distantly related protein structures: algorithm, bound and implications to homology modeling." Bioinformatics 27.18 (2011): 2537-2545.
			</div>


			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>


			<div>
					Klipcan, Liron, et al. "Structural aspects of phenylalanylation and quality control in three major forms of phenylalanyl-tRNA synthetase." Journal of amino acids 2010 (2010).
				</div>

					<div>
				Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.
			</div>
		
		</div>


	</div>
</body>
</html>
