


<html>
<head>
<title>aaRS</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="../../js/aaRS.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>




<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS("ThrRS", "Threonyl tRNA Synthetase", "../../fig/icon_polar.png");
}



</script>

</head>
<body onload="init()">


	<div id="main">

		
		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>



					Threonyl tRNA synthetase (ThrRS) is an enzyme which attaches threonine to its cognate tRNA: <br>


					<ul>
						<li>Thr + tRNA<sup>thr</sup> + ATP &rightarrow; Thr-tRNA<sup>Thr</sup> + AMP + PP<sub>i</sub> </li>
					</ul>


					Although the class II superfamily phylogeny is not entirely clear, ThrRS is part of subclass IIa alongside <a href="../his">HisRS</a>, <a href="../pro">ProRS</a>, <a href="../ser1">SerRS</a>, and the <a href="../gly1">dimeric GlyRS</a> (Gomez et al. 2020, Valencia-Sánchez et al. 2016, Perona et al. 2012). With the exception of SerRS, the anticodon binding domains of subclass IIa are homologous and reside at the C-terminal (Wolf et al. 1999). <br><br> 



					The catalytic domain of ThrRS is quite typical of a class II aaRS. Like most members of the superfamily, ATP binding is coordinated by the arginine tweezers, located in motifs 2 and 3 (Kaiser et al. 2018). The catalytic domain contains six or seven antiparallel sheets. <br><br>


					Editing in ThrRS occurs at the post-transfer level. This process has been characterised for the removal of mischarged serines from tRNA<sup>Thr</sup> (Dock-Bregeon et al. 2000). This function is achieved through the two N-termimal domains N1 and N2 (Dock-Bregeon et a. 2004). A non-homologous N-terminal editing domain (which also targets mischarged serine) is present in some archaea instead of the N1 and N2 domains (Hussain et al. 2006).
					The editing domain of ThrRS is upstream of the catalytic domain, and is homologous with that of <a href="../ala">AlaRS</a>, which conversely resides downstream of the catalytic domain (Sankaranarayanan et al. 1999). 
				
				<!--
					Interesting N-terminal domains (<a href="https://www.rcsb.org/groups/sequence/polymer_entity/Q8NW68">rcsb</a>) including one from beta-grasp superfamily (<a href="http://www.cathdb.info/version/latest/superfamily/3.10.20.30">CATH</a>). Their functions are unclear to me. C-terminal anticodon binding domain part of a superfamily alongside that of HisRS, GlyRS, and ProRS (<a href="http://www.ebi.ac.uk/interpro/entry/InterPro/IPR036621/">interpro</a>).
		
				
				-->
				
			
				</div>
				
			</li>
			
		</ul>




		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>
		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>

		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">
			

			<div>
				Wolf, Yuri I., et al. "Evolution of aminoacyl-tRNA synthetases—analysis of unique domain architectures and phylogenetic trees reveals a complex history of horizontal gene transfer events." Genome research 9.8 (1999): 689-710.
			</div>

			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>

				<div>
				Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.
			</div>

			<div>
				Torres-Larios, Alfredo, et al. "Structural basis of translational control by Escherichia coli threonyl tRNA synthetase." nature structural biology 9.5 (2002): 343-347.
			</div>

				<div>
				Dock-Bregeon, Anne-Catherine, et al. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 16: Threonyl-tRNA Synthetases.
			</div>

			<div>
				Dock-Bregeon, Anne-Catherine, et al. "Transfer RNA–mediated editing in threonyl-tRNA synthetase: the class II solution to the double discrimination problem." Cell 103.6 (2000): 877-884.
			</div>


			<div>
				Dock-Bregeon, Anne-Catherine, et al. "Achieving error-free translation: the mechanism of proofreading of threonyl-tRNA synthetase at atomic resolution." Molecular cell 16.3 (2004): 375-386.
			</div>

			<div>
				Hussain, Tanweer, et al. "Post‐transfer editing mechanism of ad‐aminoacyl‐tRNA deacylase‐like domain in threonyl‐tRNA synthetase from archaea." The EMBO Journal 25.17 (2006): 4152-4162.
			</div>

			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>


				<div>
					Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.
			</div>


						<div>
				Sankaranarayanan, Rajan, et al. "The structure of threonyl-tRNA synthetase-tRNAThr complex enlightens its repressor activity and reveals an essential zinc ion in the active site." Cell 97.3 (1999): 371-381.
			</div>



		
		</div>


	</div>
</body>
</html>
