


<html>
<head>
<title>aaRS</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="../../js/aaRS.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>




<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>
				
				<!--
					C-terminal domain is associated with editing (<a href="https://www.science.org/doi/full/10.1126/science.1174343">Guo 2009</a>, <a href="https://www.rcsb.org/structure/3G98#entity-1">rcsb</a>). 
			This domain is missing from most crystal structures but appears above in the alphafold sequences.
				-->


				AlaRS catalyses the following reaction: <br>


					<ul>
						<li>Ala + tRNA<sup>Ala</sup> + ATP &rightarrow; Ala-tRNA<sup>Ala</sup> + AMP + PP<sub>i</sub> </li>
					</ul>


					The catalytic core of AlaRS is similar to the <a href="../gly2">tetrameric GlyRS</a> found mainly in bacteria.
					Although the class II superfamily phylogeny is not entirely clear, these two members appear to be closely related, constituting subclass IIc or IId depending on the author (Valencia-Sánchez et al. 2016, Gomez et al. 2020). <br><br>


					The catalytic domain of AlaRS has seen numerous deletions and stands apart from other class II members.

					Aside from these truncations, the catalytic domain contains all seven antiparallel strands rather than seven, and like most members of the superfamily, ATP binding is coordinated by the arginine tweezers, located in motifs 2 and 3 (Kaiser et al. 2018).  <br><br>



				Editing in AlaRS occurs at both the pre-transfer and post-transfer levels. Pre-transfer editing occurs through a C-terminal editing domain, which targets mischarged glycyl and seryl groups from tRNA<sup>Ala</sup> (Tsui et al. 1981, Guo et al. 2009b). At the post-transfer level, AlaX proteins (which are homologous with he C-terminal editing domain of AlaRS) also target Ser-tRNA<sup>Ala</sup> (Beebe et al. 2008). This double checkpoint system minimises serine-for-alanline mistranslations, which are known to cause profound problems in nature (Guo et al. 2009a, Schimmel 2011). The C-terminal editing domain is homologous with the N-terminal editing domain of <a href="../thr">ThrRS</a> (Sankaranarayanan et al. 1999). 

				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">
		
			<div>
				Schimmel, Paul. "Mistranslation and its control by tRNA synthetases." Philosophical Transactions of the Royal Society B: Biological Sciences 366.1580 (2011): 2965-2971.
			</div>

			<div>
				Guo, Min, et al. "Paradox of mistranslation of serine for alanine caused by AlaRS recognition dilemma." Nature 462.7274 (2009a): 808-812.
			</div>

			<div>
				Guo, Min, et al. "The C-Ala domain brings together editing and aminoacylation functions on one tRNA." Science 325.5941 (2009b): 744-747.
			</div>


			<div>
				Beebe, Kirk, et al. "Distinct domains of tRNA synthetase recognize the same base pair." Nature 451.7174 (2008): 90-93.
			</div>

			<div>
				Sankaranarayanan, Rajan, et al. "The structure of threonyl-tRNA synthetase-tRNAThr complex enlightens its repressor activity and reveals an essential zinc ion in the active site." Cell 97.3 (1999): 371-381.
			</div>

			<div>
				Tsui, W-C., and Alan R. Fersht. "Probing the principles of amino acid selection using the alanyl-tRNA synthetase from Escherichia coli." Nucleic Acids Research 9.18 (1981): 4627-4637.
			</div>
			
			<div>
				Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.
			</div>


				<div>
				Ibba, Michael, et al. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 21: Alanyl-tRNA Synthetases.
			</div>

			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>



			<div>
				Cusack, Stephen, Michael Härtlein, and Reuben Leberman. "Sequence, structural and evolutionary relationships between class 2 aminoacyl-tRNA synthetases." Nucleic acids research 19.13 (1991): 3489-3498.
			</div>

			<div>
					Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.
			</div>

		
		
		</div>


	</div>
</body>
</html>
