


<html>
<head>
<title>Alanyl-tRNA Synthetase</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>


        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>





<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-KBC488H8EG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KBC488H8EG');
</script>


<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>
				
				<!--
					C-terminal domain is associated with editing (<a href="https://www.science.org/doi/full/10.1126/science.1174343">Guo 2009</a>, <a href="https://www.rcsb.org/structure/3G98#entity-1">rcsb</a>). 
			This domain is missing from most crystal structures but appears above in the alphafold sequences.
				-->


					Alanyl-tRNA synthetase (AlaRS) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of the amino acid alanine to its cognate tRNA:




					<div class="reaction">
						<span>
							$ \text{Ala} + \text{tRNA}^\text{Ala} + \text{ATP} \xrightarrow{\text{AlaRS}} \text{Ala-tRNA}^\text{Ala} + \text{AMP} + \text{PP}_i  $
						</span>
					</div>


					
					
					The catalytic core of AlaRS is similar to the <a href="../gly2">tetrameric GlyRS</a> found mainly in bacteria and belongs to subclass IId (Valencia-Sánchez et al. 2016). 
					The alpha helix rich tRNA binding domain primarily recognises elements within the acceptor stem of tRNA, as opposed to the anticodon (Naganuma et al. 2009).
					Although the catalytic domain of AlaRS is one of the smallest in the superfamily, the full enzyme is quite large and contains a unique domain architecture. 
					Homodimerization in AlaRS occurs via a C-terminal coiled coil, unlike most class II families which dimerise at the catalytic core  (Naganuma et al. 2009).  
					However, in certain organisms such as <i>Nanoarchaeum equitans</i>, AlaRS is produced by two genes which express the N- and C-terminal regions respectively, and thus likely behaves as a heterotetramer (Arutaki et al. 2020).
					The divergence of a homodimer into a heterotetramer has also been observed in <a href="/class2/pyl">PylRS</a>. 
					<br><br>

					The catalytic domain of AlaRS lacks a stretch of short secondary structural elements between motifs 1 and 2, which are typically found in other class II proteins. 
					The sequences at the positions of motifs 1 and 2 are also quite distant from other class members. 
					Aside from these idiosyncrasies, the catalytic domain is fairly typical of a Class II synthetease. 
					Like most members of the superfamily, ATP binding is coordinated by the arginine tweezers, located in motifs 2 and 3 (Kaiser et al. 2018).
					The bacterial and eukaryotic catalytic domain is characterised by the 30-40 residue <a href="/superfamily/class2/AlaRS_IM/">AlaRS insertion module</a> which resides between motits 2 and 3 and contains a short &beta; strand which runs parallel with the six stranded anti-parallel fold.   <br><br>

					Editing in AlaRS occurs at the post-transfer level through an editing domain which expunges mischarged glycyl and seryl groups from tRNA<sup>Ala</sup> (Tsui et al. 1981, Guo et al. 2009b).  
					AlaX proteins, which are homologous with the editing domain, also target Ser-tRNA<sup>Ala</sup> (Beebe et al. 2008). 
					This double checkpoint system minimizes serine-for-alanine mistranslations, which are known to cause profound problems in nature (Guo et al. 2009a, Schimmel 2011). 
					The C-ala domain is tethered to the editing domain through a coiled coil and promotes cooperative binding between the catalytic core and the editing domains to tRNA<sup>Ala</sup> (Guo et al. 2009b). 
					The <a href="/superfamily/class2/Editing_domain_AT/">editing domain</a> of AlaRS is downstream of the catalytic domain and is homologous with that of <a href="../thr">ThrRS</a>, which conversely resides upstream of the catalytic domain (Sankaranarayanan et al. 1999).  <br><br>
										
										


				
					In eukaryotes,  AlaRS is one of the few aminoacyl-tRNA synthetases which operate in both the cytoplasm as well as the mitochondria, alongside <a href="/class2/gly3">GlyRS-E</a>, <a href="/class2/his">HisRS</a>, and <a href="/class1/val">ValRS</a>.
					Localisation into these compartments is achieved through alternative initiation, which governs the expression of an N-terminal mitochondrial localisation signal  (Tang et al. 2004). 
					One exception to this is the AlaRS of the yeast <i>Vanderwaltozyma polyspora</i>, which has two genes encoding AlaRS; one for either compartment (Chang et al. 2011). 
					It is likely that these two genes arose from a recent duplication event. 
					
				



				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">
		
			<div>
				Schimmel, Paul. "Mistranslation and its control by tRNA synthetases." Philosophical Transactions of the Royal Society B: Biological Sciences 366.1580 (2011): 2965-2971.
			</div>

			<div>
				Guo, Min, et al. "Paradox of mistranslation of serine for alanine caused by AlaRS recognition dilemma." Nature 462.7274 (2009a): 808-812.
			</div>

			<div>
				Guo, Min, et al. "The C-Ala domain brings together editing and aminoacylation functions on one tRNA." Science 325.5941 (2009b): 744-747.
			</div>


			<div>
				Tang, Huei-Lin, et al. "Translation of a yeast mitochondrial tRNA synthetase initiated at redundant non-AUG codons." Journal of Biological Chemistry 279.48 (2004): 49656-49663.
			</div>


			<div>
				Naganuma, Masahiro, et al. "Unique protein architecture of alanyl-tRNA synthetase for aminoacylation, editing, and dimerization." Proceedings of the National Academy of Sciences 106.21 (2009): 8489-8494.
			</div>

			<div>
				Chang, Chia-Pei, et al. "Alanyl-tRNA synthetase genes of Vanderwaltozyma polyspora arose from duplication of a dual-functional predecessor of mitochondrial origin." Nucleic acids research 40.1 (2012): 314-322.
			</div>

			<div>
				Beebe, Kirk, et al. "Distinct domains of tRNA synthetase recognize the same base pair." Nature 451.7174 (2008): 90-93.
			</div>

			<div>
				Sankaranarayanan, Rajan, et al. "The structure of threonyl-tRNA synthetase-tRNAThr complex enlightens its repressor activity and reveals an essential zinc ion in the active site." Cell 97.3 (1999): 371-381.
			</div>

			<div>
				Tsui, W-C., and Alan R. Fersht. "Probing the principles of amino acid selection using the alanyl-tRNA synthetase from Escherichia coli." Nucleic Acids Research 9.18 (1981): 4627-4637.
			</div>
			
			<div>
				Kaiser, Florian, et al. "Backbone brackets and arginine tweezers delineate class I and class II aminoacyl tRNA synthetases." PLoS computational biology 14.4 (2018): e1006101.
			</div>


			<div>
				de Pouplana, Lluis Ribas, et al. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 21: Alanyl-tRNA Synthetases.
			</div>

			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>

			<div>
				Cusack, Stephen, Michael Härtlein, and Reuben Leberman. "Sequence, structural and evolutionary relationships between class 2 aminoacyl-tRNA synthetases." Nucleic acids research 19.13 (1991): 3489-3498.
			</div>

			<div>
					Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.
			</div>
			
			<div>
				Arutaki, Misa, et al. "G: U-Independent RNA minihelix aminoacylation by Nanoarchaeum equitans alanyl-tRNA synthetase: an insight into the evolution of aminoacyl-tRNA synthetases." Journal of molecular evolution 88 (2020): 501-509.
			</div>

		
		
		</div>


	</div>
</body>
</html>
