


<html>
<head>
<title>Tryptophanyl-tRNA Synthetase</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-KBC488H8EG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KBC488H8EG');
</script>


<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>
				
					
						Tryptophanyl-tRNA synthetase (TrpRS) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of tyrosine to its cognate tRNA:


					<div class="reaction">
						<span>

							$ \text{Trp} + \text{tRNA}^\text{TrpRS} + \text{ATP} \xrightarrow{\text{TrpRS}} \text{Trp-tRNA}^\text{TrpRS} + \text{AMP} + \text{PP}_i  $

						</span>
					</div>
				


					TrpRS shares a number of similarites with its close homolog <a href="/class1/tyr">TyrRS</a>, which constitute subclass Ic.
					TyrRS and TrpRS are homodimeric, in contrast to many Class I syntheteases which operate as monomers (Gomez and Ibba, 2020).
					Moreover, the two amino acids may have been among the last amino acids to emerge in the metabolism (Trifonov 2000), 
					and are chemically similar (aromatic side chains), consistent with the hypotheses of the two AARS sharing a common ancestor with either dual specificity, or one synthetase spawning off the other (Doublié et al. 1995, Fournier et al. 2015).
					Conserved aspartate side chains in the specificity determining helix (SPH) allow the respective enzymes to discriminate between tyrosine and tryptophan (Doublié et al. 1995).
					Lastly, while most Class I syntheteases approach the tRNA acceptor stem from the minor groove side, TyrRS and TrpRS alike instead approach the acceptor stem directly (de Pouplana and Schimmel 2001).
					<br><br>
					
					
					TrpRS from prokaryotic and archaeal/eukaryotic organisms form quite distinct groups, with respect to their sequences, allosteries, and domain architectures (Carter 2005).
					The archaeal-like TrpRS contains an additional N-terminal domain known as the eukaryote-specific-extension, ESE (Yang et al. 2007).
					Both the ESE and C-terminal anticodon binding domains undergo vast conformational changes upon substrate binding, making the archaeal-like form more complex than the bacterial-like (Shen et al. 2008).
					However, despite the intricate allostery displayed by TrpRS, the catalytic domain itself is less elaborated than other Class I synthetases, making it a useful target for investigating
					ancestral models, such as  protozymes (Onodera et al. 2021) and urzymes (Pham et al. 2010), which display activation and aminoacylation activity respectively. 
					The connecting peptide 1 (CP1) domains in TrpRS and TyrRS occur in their simplest forms.
					<br><br>
					
					
					
					Some eukaryotes contain an additional domain upstream of the ESE.
					In humans, TrpRS contains the WHEP domain, which plays a role in the primary defense system against infection (Ahn et al. 2017).
					In certain parasites, TrpRS possesses an N-terminal domain homologous to the editing domain of 
					<a href="/class2/thr">ThrRS</a> and <a href="/class2/ala">AlaRS</a> (Merritt et al. 2011).
					
			
					


				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">
	
		
			<div>
				Charles W Carter. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 11: Tryptophanyl-tRNA Synthetases.
			</div>
			
			
			<div>
				Doublié, Sylvie, et al. "Tryptophanyl-tRNA synthetase crystal structure reveals an unexpected homology to tyrosyl-tRNA synthetase." Structure 3.1 (1995): 17-31.
			</div>
	
	
			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." RNA 26.8 (2020): 910-936.
			</div>
			
			
			<div>
				de Pouplana, Lluı́s Ribas, and Paul Schimmel. "Aminoacyl-tRNA synthetases: potential markers of genetic code development." Trends in biochemical sciences 26.10 (2001): 591-596.
			</div>
			
			<div>
				Dong, Xianchi, et al. "Crystal structure of Pyrococcus horikoshii tryptophanyl-tRNA synthetase and structure-based phylogenetic analysis suggest an archaeal origin of tryptophanyl-tRNA synthetase." Nucleic acids research 38.4 (2010): 1401-1412.
			</div>
			
			<div>
				Zhou, Minyun, et al. "Crystal structures of Saccharomyces cerevisiae tryptophanyl-tRNA synthetase: new insights into the mechanism of tryptophan activation and implications for anti-fungal drug design." Nucleic acids research 38.10 (2010): 3399-3413.
			</div>
			
			<div>
				Trifonov, Edward N. "Consensus temporal order of amino acids and evolution of the triplet code." Gene 261.1 (2000): 139-151.
			</div>
			
			<div>
				Shen, Ning, et al. "Catalytic mechanism of the tryptophan activation reaction revealed by crystal structures of human tryptophanyl-tRNA synthetase in different enzymatic states." Nucleic acids research 36.4 (2008): 1288-1299.
			</div>
			
			<div>
				Ahn, Young Ha, et al. "Secreted tryptophanyl-tRNA synthetase as a primary defence system against infection." Nature microbiology 2.1 (2016): 1-13.
			</div>
			
			<div>
				Merritt, Ethan A., et al. "Crystal structures of three protozoan homologs of tryptophanyl-tRNA synthetase." Molecular and biochemical parasitology 177.1 (2011): 20-28.
			</div>
			
			<div>
				Yang, Xiang-Lei, et al. "Functional and crystal structure analysis of active site adaptations of a potent anti-angiogenic human tRNA synthetase." Structure 15.7 (2007): 793-805.
			</div>
			
			<div>
			Pham, Yen, et al. "Tryptophanyl-tRNA synthetase Urzyme: a model to recapitulate molecular evolution and investigate intramolecular complementation." Journal of Biological Chemistry 285.49 (2010): 38590-38601.
			</div>
			
			<div>
				Onodera, Kazaha, et al. "Amino acid activation analysis of primitive aminoacyl-tRNA synthetases encoded by both strands of a single gene using the malachite green assay." BioSystems 208 (2021): 104481.
			</div>

		
		</div>


	</div>
</body>
</html>
