


<html>
<head>
<title>Leucyl-tRNA Synthetase (Bacterial-like)</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>

<script async src="https://www.googletagmanager.com/gtag/js?id=G-KBC488H8EG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KBC488H8EG');
</script>



<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>
				


					The bacterial-like leucyl-tRNA synthetase (LeuRS-B) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of the amino acid leucine to its cognate tRNA:




					<div class="reaction">
						<span>
							$ \text{Leu} + \text{tRNA}^\text{Leu} + \text{ATP} \xrightarrow{\text{LeuRS-B}} \text{Leu-tRNA}^\text{Leu} + \text{AMP} + \text{PP}_i  $
						</span>
					</div>




							LeuRS-B is found in most bacteria and organelles, and differs from the archaeal-like form LeuRS-A, which is found in most archaea. 
							Either type may reside in a eukaryotic cytosol.  
							The two forms are characterised by their abilities to charge tRNAs corresponding to six different codons.
							In the standard genetic code, <a href="/class1/arg">ArgRS</a> and <a href="/class2/ser1">SerRS</a> are the only other AARS which decode as many as six codons.
							The structures of LeuRS-A and -B are distinguished by 
							i) the placement of the editing domain within their primary sequence  (Fukunaga et al. 2005), 
							ii) the C-terminal domain which recognises tRNA (Tukalo et al. 2005), and 
							iii) the LeuRS-B insertion module below, also known as the leucyl-specific domain (Cusack et al. 2000). <br><br>


							LeuRS-A and -B alike are closely related to the <a href="../val">ValRS</a>, <a href="../ile">IleRS</a>, and <a href="../met">MetRS</a> families, which comprise 
							subclass Ia (Perona and Hadd 2012, Gomez and Ibba, 2020).
							Members of subclass Ia are characterised by their hydrophobic amino acid substrates, as well as the connecting peptide 2 (CP2) and zinc finger insertions, depicted below. 
						    CP2 is 30-40 amino acid two-helix bundle on the surface of the catalytic domain (Starzyk et al. 1987) and appears to be essential for amino acid activation (Zhou at al. 2008). 
							Upstream from this resides a cysteine-rich zinc finger, 20-40 amino acids in length, also essential for effective aminoacylation (Nureki et al. 1993, Sugiura et al. 2000). <br><br>


							ValRS,	IleRS, and LeuRS share a a post-transfer editing domain, absent from MetRS.
							This editing domain typically resides within the zinc finger, providing further amino acid selectivity by expelling a wide range of mistargetted amino acids 
							such alanine, cysteine, threonine, valine, isoleucine, methionine, homocysteine, and norvaline (Gomez and Ibba, 2020).
							However, unlike LeuRS-A, ValRS, and IleRS, the editing domain of LeuRS-B resides downstream of the zinc finger and CP2 (Fukunaga et al. 2005).
							Thus, the domain appears to have hopped between the two positions, or may have orginated as a trans-acting factor.
							IleRS, ValRS, and LeuRS-B are further characterised by connecting peptide 3 (CP3), which contains a second zinc-finger motif in certain organisms (Fukunaga et al. 2005).
							Its functional role is unclear. <br><br>




							The tRNA is recognised by two C-terminal domains (Tukalo et al. 2005, Rock et al. 2007).
							The first is a <a href="/superfamily/class1/Anticodon_binding_domain_1a">helical domain</a> that recognises the anticodon binding domain, and is universal across the members of subclass Ia plus <a href="../cys">CysRS</a> and <a href="../arg">ArgRS</a>.
							The second is a compact $\alpha$-$\beta$ domain unique to LeuRS-B.
							The domain is flexibly linked with the rest of the protein and mutational studies suggest it is essential for effective aminoacylation (Tukalo et al. 2005).
							 <br><br>







				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">


			<div>
				Tommie L. Lincecum, Jr. and Susan A. Martinis. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 5: Leucyl-tRNA Synthetases.
			</div>
	
			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>

			<div>
				Cusack, Stephen, Anna Yaremchuk, and Michael Tukalo. "The 2 Å crystal structure of leucyl-tRNA synthetase and its complex with a leucyl-adenylate analogue." The EMBO Journal 19.10 (2000): 2351-2361.
			</div>


			<div>
				Zhou, Xiao-Long, Bin Zhu, and En-Duo Wang. "The CP2 domain of leucyl-tRNA synthetase is crucial for amino acid activation and post-transfer editing." Journal of Biological Chemistry 283.52 (2008): 36608-36616.
			</div>

			<div>
				Starzyk, Ruth M., Teresa A. Webster, and Paul Schimmel. "Evidence for dispensable sequences inserted into a nucleotide fold." Science 237.4822 (1987): 1614-1618.
			</div>
			
			
			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>


		<div>
				Nureki, O., et al. "Chemical modification and mutagenesis studies on zinc binding of aminoacyl-tRNA synthetases." Journal of Biological Chemistry 268.21 (1993): 15368-15373.
			</div>


	
			<div>
				Sugiura, Ikuko, et al. "The 2.0 Å crystal structure of Thermus thermophilus methionyl-tRNA synthetase reveals two RNA-binding modules." Structure 8.2 (2000): 197-208.
			</div>

			<div>
				Fukunaga, Ryuya, and Shigeyuki Yokoyama. "Crystal structure of leucyl-tRNA synthetase from the archaeon Pyrococcus horikoshii reveals a novel editing domain orientation." Journal of molecular biology 346.1 (2005): 57-71.
			</div>

			<div>
				Rock, Fernando L., et al. "An antifungal agent inhibits an aminoacyl-tRNA synthetase by trapping tRNA in the editing site." science 316.5832 (2007): 1759-1761.
			</div>

			<div>
				Tukalo, Michael, et al. "The crystal structure of leucyl-tRNA synthetase complexed with tRNALeu in the post-transfer–editing conformation." Nature structural & molecular biology 12.10 (2005): 923-930.
			</div>
		
		</div>


	</div>
</body>
</html>
