


<html>
<head>
<title>Valyl-tRNA Synthetase</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
        
        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-KBC488H8EG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KBC488H8EG');
</script>



<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>




<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">


		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">
				
				<div>



					Valyl-tRNA synthetase (ValRS) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of the amino acid valine to its cognate tRNA:




					<div class="reaction">
						<span>
							$ \text{Val} + \text{tRNA}^\text{Val} + \text{ATP} \xrightarrow{\text{ValRS}} \text{Val-tRNA}^\text{Val} + \text{AMP} + \text{PP}_i  $
						</span>
					</div>


							ValRS is closely related to the <a href="../leu2">LeuRS-A</a>, <a href="../leu1">LeuRS-B</a>, <a href="../ile">IleRS</a>, and <a href="../met">MetRS</a> families, which comprise 
							subclass Ia (Perona and Hadd 2012, Gomez and Ibba, 2020).
							Members of subclass Ia are characterised by their hydrophobic amino acid substrates, as well as the connecting peptide 2 (CP2) and zinc finger insertions, depicted below. 
						    CP2 is a 30-40 amino acid two-helix bundle on the surface of the catalytic domain (Starzyk et al. 1987) and appears to be essential for amino acid activation (Zhou at al. 2008). 
							Upstream from this resides a cysteine-rich zinc finger, 20-40 amino acids in length, also essential for effective aminoacylation (Nureki et al. 1993, Sugiura et al. 2000). <br><br>


							ValRS,	IleRS, and LeuRS share a a post-transfer editing domain, absent from MetRS.
							This editing domain typically resides within the zinc finger, providing further amino acid selectivity by expelling a wide range of mistargetted amino acids 
							such alanine, cysteine, threonine, valine, isoleucine, methionine, homocysteine, and norvaline (Gomez and Ibba, 2020).
							ValRS also targets misactivated amino acids, such as threonine, through pre-transfer editing activity (Fersht et al. 1976). 
							Its discrimination first against size (i.e. by filtering out isoleucine), and second against polarity (i.e. removal of misactivated threonine) has been described as a double-sieve mechanism (Fersht et al. 1976, Fukai et al. 2000).
							IleRS, ValRS, and the bacterial-like LeuRS-B are further characterised by connecting peptide 3 (CP3), which contains a second zinc-finger motif in certain organisms (Fukunaga et al. 2005).
							Its functional role is unclear. <br><br>



							The anitcodon loop is recognised by three C-terminal domains.
							The first is a <a href="/superfamily/class1/Anticodon_binding_domain_1a">helical domain</a>, universal across the members of subclass Ia, as well as <a href="../cys">CysRS</a> and <a href="../arg">ArgRS</a>.
							The second is a $\alpha$/$\beta$ domain which recognises the anticodon stem (Fukai et al. 2000).
							Lastly, a coiled coil recognises the D-loop of the tRNA (Fukai et al. 2000), however this domain is absent in archaea. <br><br>


					
								ValRS is one of the few aminoacyl-tRNA synthetases which operate in both the cytoplasm as well as the mitochondria, alongside <a href="/class2/ala">AlaRS</a>, <a href="/class2/gly3">GlyRS-E</a>, and <a href="/class2/his">HisRS</a> (Chatton et al. 1986).
								Phylogenetic analyses sugsest that, in eukaryotes, the cytosolic ValRS was transferred from the mitochondrial form, and is therefore of bacterial origin (Brown et a. 1995).
								Thus, the archaeal form of ValRS stands apart from the rest of the family.
					

				</div>
				
			</li>
			
		</ul>
		

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		
		
		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>


		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		<div id="references">


			<div>
				Osamu Nureki and Shigeyuki Yokoyama. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 7: Valyl-tRNA Synthetases.
			</div>
	
			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>

			<div>
				Chatton, Bruno, et al. "The yeast VAS1 gene encodes both mitochondrial and cytoplasmic valyl-tRNA synthetases." Journal of Biological Chemistry 263.1 (1988): 52-57.
			</div>


			<div>
				Brown, James R., and W. Ford Doolittle. "Root of the universal tree of life based on ancient aminoacyl-tRNA synthetase gene duplications." Proceedings of the National Academy of Sciences 92.7 (1995): 2441-2445.
			</div>


			<div>
				Fersht, Alan R., and Meredith M. Kaethner. "Enzyme hyperspecificity. Rejection of threonine by the valyl-tRNA synthetase by misacylation and hydrolytic editing." Biochemistry 15.15 (1976): 3342-3346.
			</div>

			<div>
				Zhou, Xiao-Long, Bin Zhu, and En-Duo Wang. "The CP2 domain of leucyl-tRNA synthetase is crucial for amino acid activation and post-transfer editing." Journal of Biological Chemistry 283.52 (2008): 36608-36616.
			</div>

			<div>
				Starzyk, Ruth M., Teresa A. Webster, and Paul Schimmel. "Evidence for dispensable sequences inserted into a nucleotide fold." Science 237.4822 (1987): 1614-1618.
			</div>
			
			
			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>


		<div>
				Nureki, O., et al. "Chemical modification and mutagenesis studies on zinc binding of aminoacyl-tRNA synthetases." Journal of Biological Chemistry 268.21 (1993): 15368-15373.
			</div>


	
			<div>
				Sugiura, Ikuko, et al. "The 2.0 Å crystal structure of Thermus thermophilus methionyl-tRNA synthetase reveals two RNA-binding modules." Structure 8.2 (2000): 197-208.
			</div>

			<div>
				Fukunaga, Ryuya, and Shigeyuki Yokoyama. "Crystal structure of leucyl-tRNA synthetase from the archaeon Pyrococcus horikoshii reveals a novel editing domain orientation." Journal of molecular biology 346.1 (2005): 57-71.
			</div>


			<div>

				Fukai, Shuya, et al. "Structural basis for double-sieve discrimination of L-valine from L-isoleucine and L-threonine by the complex of tRNAVal and valyl-tRNA synthetase." Cell 103.5 (2000): 793-803.
			</div>
		
		</div>


	</div>
</body>
</html>
