


<html>
<head>
<title>Phenylalanyl-tRNA Synthetase &beta; Subunit (Bacterial-like)</title>

<link rel="icon" >
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../../css/styles.css">

<script language='JavaScript' type='text/JavaScript' src="/js/aaRS.js"></script>
<script language='JavaScript' type='text/JavaScript' src="/js/util.js"></script>
<script type='text/javascript' src='../../bower_components/bio-pv/bio-pv.min.js'></script>

        
<script type="text/javascript"
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true
  }
});
</script>





<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>

<script async src="https://www.googletagmanager.com/gtag/js?id=G-KBC488H8EG"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KBC488H8EG');
</script>



<script language='JavaScript' type='text/JavaScript'>


function init(){
	
	renderaaRS();
}



</script>

</head>
<body onload="init()">


	<div id="main">

		<ul id="flexContainer">
			<li class="summary">
				
			</li>


			<li class="notes">





					The bacterial-like phenylalanyl-tRNA synthetase (PheRS-B) is an enzyme that plays a crucial role in protein synthesis by catalysing the attachment of the amino acid phenylalanine to its cognate tRNA:



					<div class="reaction">
						<span>
							$ \text{Phe} + \text{tRNA}^\text{Phe} + \text{ATP} \xrightarrow{\text{PheRS-B}} \text{Phe-tRNA}^\text{Phe} + \text{AMP} + \text{PP}_i  $
						</span>
					</div>




				The phenylalanyl-tRNA synthetases (PheRS) are considered to be one of the most evolutionarily complex enzymes among aminoacyl-tRNA synthetases (Klipcan et al. 2010).
					Unlike most members of the class II superfamily, which are homodimeric, PheRS-B operates as a heterotetramer composed of two &alpha; and two &beta; subunits &alpha;<sub>2</sub>&beta;<sub>2</sub>. 
					There are three known variants of PheRS: one found predominantly in bacteria, one found in eukaryotes/archaea (<a href="../phe3">PheRS-A</a>), and one localsied to eukaryotic mitochondria (<a href="../phe3">PheRS-M</a>). 
					This page specifically refers to the &beta; subunit of the bacterial variant (PheRS-B&beta;). <br><br>

					The catalytic activity of PheRS-B is restricted to the &alpha; subunit, while tRNA recognition and editing activity is restricted to the &beta; subunit, 
					which is over twice the size of the &alpha; subunit (Fishman et al. 2001). The only other tetrameric member of class II is the bacterial 
					<a href="../gly2">GlyRS</a>, which is also composed of two catalytic &alpha; subunits and two non-catalytic &beta; subunits. 
					However, unlike GlyRS, catalysis in PheRS-B is not possible without the full tetrameric assembly (Mosyak et al. 1995). 
					PheRS-B binds two tRNA molecules, with each tRNA binding across all four subunits (Goldgur et al. 1997). <br><br>

					
					The C-terminal catalytic core of the &beta; subunit is similar to that of other PheRS families, as well as <a href="../sep">SepRS</a>, and belongs to subclass IIc
					(Kavran et al. 2007, Perona et al. 2012, Valencia-Sánchez et al. 2016, Gomez et al. 2020). 
					The catalytic domain does not possess catalytic activity (Fishman et al. 2001), but still contains the six antiparallel strands characteristic of a class II aaRS. 
					As such, the sequence at the stuctural positions of motifs 1, 2, and 3 do not follow the motif pattern, but the positions one would normally expect to see these motifs are 
					labelled for convenience. <br><br>


					
					The &beta; subunit's editing domain facilitates the removal of mischarged tyrosyl groups from tRNA<sup>Phe</sup>.
					The editing domain is located at domains B3-B4, near the N-terminus (Roy et al. 2004, Mosyak et al. 1995). 
					The anticodon binding domain (domain B8) is located at the C-terminus. This domain is unique in its charging of the 2′OH of tRNA, 
					while other members of class II charge 3′OH (Goldgur et al. 1997).
					This contrasts with the eukaryotic PheRS tetramer, whose anticodon binding domain resides in the &alpha; chain.
					A paralog of the N-terminal RNA binding domain EMAP is also found in some forms of <a href="/class1/tyr">TyrRS</a> and <a href="/class1/met">MetRS</a> (Wolf et al. 1999).
					<br><br>




				
				<div>
			
				</div>
				
			</li>
			
		</ul>

		<div class="svgDiv">
			<svg id="secondary" height=0 width=0 overflow="auto"></svg>
		</div>

		<div id="alignment" class="svgDiv">

		</div>


		<div id="alignment2" class="svgDiv">

		</div>




		<div id="tertiaryTable">
			
			<table>
				<tr>
					<td>
						<div id="superposition"> </div>
					</td>
					
					<td>
						<div id="tertiary"> </div>
					</td>
				</tr>
			
			</table>
			
			
		
		</div>
		
		
		
			<div id="references">


			<div>
					Klipcan, Liron, et al. "Structural aspects of phenylalanylation and quality control in three major forms of phenylalanyl-tRNA synthetase." Journal of amino acids 2010 (2010).
				</div>


				<div>
					Roy, Herve, et al. "Post‐transfer editing in vitro and in vivo by the β subunit of phenylalanyl‐tRNA synthetase." The EMBO journal 23.23 (2004): 4639-4648.
				</div>

				<div>
					Finarov, Igal, et al. "Structure of human cytosolic phenylalanyl-tRNA synthetase: evidence for kingdom-specific design of the active sites and tRNA binding patterns." Structure 18.3 (2010): 343-353.
				</div>

				<div>
					Mosyak, Lidia, et al. "Structure of phenylalanyl-tRNA synthetase from Thermus thermophilus." Nature structural biology 2.7 (1995): 537-547.
				</div>

				<div>
					Goldgur, Yehuda, et al. "The crystal structure of phenylalanyl-tRNA synthetase from Thermus thermophilus complexed with cognate tRNAPhe." Structure 5.1 (1997): 59-68.
				</div>

				<div>
					Fishman, Roman, et al. "Structure at 2.6 Å resolution of phenylalanyl-tRNA synthetase complexed with phenylalanyl-adenylate in the presence of manganese." Acta Crystallographica Section D: Biological Crystallography 57.11 (2001): 1534-1544.
				</div>


							<div>
				Kavran, Jennifer M., et al. "Structure of pyrrolysyl-tRNA synthetase, an archaeal enzyme for genetic code innovation." Proceedings of the National Academy of Sciences 104.27 (2007): 11268-11273.
			</div>

			<div>
				Cusack, Stephen, Michael Härtlein, and Reuben Leberman. "Sequence, structural and evolutionary relationships between class 2 aminoacyl-tRNA synthetases." Nucleic acids research 19.13 (1991): 3489-3498.
			</div>

			<div>
					Valencia-Sánchez, Marco Igor, et al. "Structural Insights into the Polyphyletic Origins of Glycyl tRNA Synthetases." Journal of Biological Chemistry 291.28 (2016): 14430-14446.
			</div>


				<div>
				Safro, Mark. "The Aminoacyl-tRNA Synthetases" CRC Press (2005): Chapter 22: Phenylalanyl-tRNA Synthetases.
			</div>

			<div>
				Gomez, Miguel Angel Rubio, and Michael Ibba. "Aminoacyl-tRNA synthetases." Rna 26.8 (2020): 910-936.
			</div>


			<div>
				Perona, John J., and Andrew Hadd. "Structural diversity and protein engineering of the aminoacyl-tRNA synthetases." Biochemistry 51.44 (2012): 8705-8729.
			</div>
			
			
			<div>
				Wolf, Yuri I., et al. "Evolution of aminoacyl-tRNA synthetases—analysis of unique domain architectures and phylogenetic trees reveals a complex history of horizontal gene transfer events." Genome research 9.8 (1999): 689-710.
			</div>


		
		</div>




	</div>
</body>
</html>
